docker-compose.override.yml

```diff
@@ -9,8 +9,11 @@ services:
       - "traefik.http.routers.${COMPOSE_PROJECT_NAME}.middlewares=redirect-to-https"
       - "traefik.http.middlewares.redirect-to-https.redirectscheme.scheme=https"
 
+  phpfpm:
+    image: itkdev/php8.3-fpm:latest
```

Comment:
Should not be needed in override, templates almost updated: https://github.com/itk-dev/devops_itkdev-docker/pull/102